package.json

```diff
@@ -116,14 +108,22 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-mocha": "^9.0.0",
     "eslint-plugin-promise": "^6.0.0",
+    "husky": "^8.0.3",
     "lint-staged": "^13.0.3",
     "mocha": "^10.0.0",
-    "pre-commit": "^1.1.3",
+    "prettier": "^2.8.8",
     "rimraf": "^5.0.0",
     "semantic-release": "^20.0.2",
     "sinon": "^15.0.0",
     "ts-node": "^10.9.1",
-    "typescript": "^5.0.4",
+    "typescript": "^4.9.5",
     "xpath": "^0.x"
+  },
+  "peerDependencies": {
```

Comment:
I was not quite sure about this. This is a library rather than standalone driver, so it makes more sense to import versioned server dependencies rather than use peer ones.

Reply:
hmm I think we still want to keep it as a peer since ultimately we want this package to still use whatever appium server the user is running, as with a driver.